site/en/docs/privacy-sandbox/chips/index.md

```diff
@@ -125,7 +125,17 @@ key={("https", "retail.example"),
      ("support.chat.example")}
 ```
 
-#### Security design
+#### First-Party Sets and cookie partitioning
+
+[First-Party Sets](blog/first-party-sets-sameparty/#how-to-define-a-first-party-set) allow related domain names that are owned and operated by the same entity to be treated as first-party in situations where first party and third party are otherwise treated differently.
+
+When an embedded service sets a cookie on a site that's a member of a First-Party Set, that cookie will be accessible to the service when it's embedded on any of the First-Party Set member sites. 
```

Comment:
How about adding "This allows the embedded service to provide a seamless, unified user session across all member sites."